src/dataman/ResultSchemaProvider.cpp

```diff
@@ -46,7 +46,8 @@ bool ResultSchemaProvider::ResultSchemaField::isValid() const {
  *
  **********************************/
 ResultSchemaProvider::ResultSchemaProvider(Schema schema)
-        : columns_(std::move(schema.get_columns())) {
+        : columns_(std::move(schema.get_columns())),
+          schemaProp_(std::move(schema.get_schema_prop())) {
```

Comment:
Result schema is used for Result.  Please don't put schema props here.

Reply:
@dangleptr ,   👍👍
Yeah, in the initial design, desc tag/edge not only displays column information, but also shows schema attributes.

This way is now abandoned. The desc tag/edge only displays column information.

Show create tag/edge not only displays column information, but also shows the schema attributes.(PR #496 )

Today I am going to update this PR, thx.

Reply:
@dangleptr ,  ResultSchemaProvider used in storage is used to read filter. So we need  schema props in ResultSchemaProvider

Reply:
Use NebulaSchemaProvider,  ResultSchemaProvider is designed for result.

Reply:
👍，Thank you very much for the guidance offline. I will fix it.